README.md

```diff
@@ -121,7 +126,8 @@ a(); // Object -- so far so good.
 a().getA(); // "a"
 ```
 
-Yes. Got it. In both of these instances, we actually created a brand new object, and then immediately threw it away, because we didn't assign it to anything. Don't worry about that.
+Yes. Got it. In both of these instances, we actually created a brand new object, and then immediately threw it away, 
+because we didn't assign it to anything. Don't worry about that.
```

Comment:
Why did you insert a line break here? All viewers that aren't brain-dead automatically wrap lines based on the window viewport width. Inserting arbitrary line breaks just makes the doc look funny when you view it in plaintext with different window widths...